src/main/webapp/enroll.html

```diff
@@ -72,18 +103,31 @@ <h2>SDK Library Detail</h2>
             <div class="form-group">
             <label for="nv">Release Notes CL</label>
             <input type="text" class="form-control" id="rnc" name="release">
+
             </div>
             
-            <button type="submit" class="btn btn-primary">ENROLL</button>
+            <div class="panel panel-primary setup-content" id="step-4">
+                <div class="panel-heading">
+                    <h3 class="panel-title"> Link to Release CL Notes</h3>
```

Comment:
Change "Release CL Notes" to "Release Notes CL"

Reply:
Done.